ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -150,3 +153,17 @@ Verify Data Science Parameter From A Duplicated Run Are The Same From The Compil
     # look for spec.params inside ${PIPELINE_TEST_FILEPATH} source code
     Should Contain    ${input_parameters}    model_obc
     Should Contain    ${input_parameters}    iris-model
+
+Configure Test For Disconnected Cluster
```

Comment:
In fact, we do not configure it specifically for a disconnected env - we just set PIP mirror.

Reply:
I've refactored the code to just talk about pip_index_url, which will be used for connected and disconnected clusters